changelogs/head.asciidoc

```diff
@@ -24,3 +24,4 @@ https://github.com/elastic/apm-server/compare/8.4\...main[View commits]
 - Added support for OpenTelemetry summary metrics {pull}7772[7772]
 - Upgraded bundled APM Java agent attacher CLI to version 1.32.0, which supports the `latest` version tag {pull}8374[8374]
 - Added `span.name` to `service_destination` metrics {pull}8391[8391]
+- Update APM integration description {pull}8592[8592]
```

Comment:
In this case the changelog should go in `apmpackage/apm/changelog.yml`. Since we're going to release another 8.3 patch, I think we should add it to the backport on the 8.3 branch.

Reply:
+1 on adding it to `8.3` changelogs & backport (the Kibana changes are also backported). You can add a label `backport-8.3` and remove the `skip-backport` label for this. 

The changelog entry should be in the `changelogs/8.3` and the `apmpackage/apm/changelog.yml`. The `changelogs/*` entries are used for https://www.elastic.co/guide/en/apm/guide/current/release-notes-8.3.html and the `apmpackage/apm/changelog.yml` is what is showing up directly in the package changelogs.

Reply:
> In this case the changelog should go in apmpackage/apm/changelog.yml. Since we're going to release another 8.3 patch, I think we should add it to the backport on the 8.3 branch.

Added :+1: 

> +1 on adding it to 8.3 changelogs & backport (the Kibana changes are also backported). You can add a label backport-8.3 and remove the skip-backport label for this.

Done, thanks for the tip! :bow: 

> The changelog entry should be in the changelogs/8.3 and the apmpackage/apm/changelog.yml.

I think I'm missing something, should I manually add the entry to `changelogs/8.3` ? I don't see a section for the next patch version

Reply:
> I think I'm missing something, should I manually add the entry to changelogs/8.3 ? I don't see a section for the next patch version

We manually create it with the first PR that adds a fix, so please just copy&paste the headers from the previous patch version and then add the bugfix.

Reply:
Ah I see, added!

Thank you for the help! :bow: